functions/src/routers/groupRouter.ts

```diff
@@ -122,3 +124,89 @@ groupRouter.post("/add", async (req, res) => {
   // }
   return;
 });
+groupRouter.get("/:userId/groups", async (req, res) => {
+  const uid = req.decodedToken.uid;
+
+  try {
+    // 데이터베이스에서 해당 유저의 정보 가져오기
+    const userData = (
+      await admin.firestore().collection("users").doc(uid).get()
+    ).data();
+
+    // console.log(uid, userData);
+
+    // DB 에서 해당 데이터를 찾을 수 없는경우.
+    if (!userData)
+      return res.status(404).json({ status: 404, error: "USERDATA NOT FOUND" });
+
+    // 해당 유저 그룹에 그룹 정보가 있는지 확인
+    if (!userData.group)
+      return res
+        .status(403)
+        .json({ status: 403, error: "USERGROUP FORBIDDEN", uid });
+
+    // 유저의 그룹 정보 데이터 반환
+    return res.json({
+      status: "200",
+      data: userData.group, // Group ID
+    });
+  } catch (e) {
+    console.error(e);
+    return res.status(500).json({ status: 500, error: "Server error", e });
+  }
+});
+
+groupRouter.post("/:groupId/add/announce", async (req, res) => {
+  const groupId = req.params.groupId.toLowerCase();
+  const {
+    announceName,
+    description,
+    wayofannounce = "email",
+    checked,
+    today,
+  } = req.body;
+  const { uid } = req.decodedToken;
+
+  // 유효성 #1 - 해당 필드들이 존재하여야함.
+  if (!announceName || !description || !wayofannounce || !checked || !today) {
```

Comment:
서버에서 날짜를 추가하기에 데이터를 받을 필요 없다.

```suggestion
  if (!announceName || !description || !wayofannounce || !checked) {
```